Source/WebCore/platform/graphics/BitmapImageDescriptor.cpp

```diff
@@ -32,8 +32,8 @@
 
 namespace WebCore {
 
-BitmapImageDescriptor::BitmapImageDescriptor(BitmapImageSource& source)
-    : m_source(source)
+BitmapImageDescriptor::BitmapImageDescriptor(CheckedRef<BitmapImageSource>&& source)
```

Comment:
We usually do not pass `CheckedRef` as parameter, this should just be a `BitmapImageSource&`